configuration/package_manifests/nfpm.c8y-configuration-plugin.yaml

```diff
@@ -28,6 +28,10 @@ deb:
     - c8y_configuration_plugin (<= 0.8.1)
 
 contents:
+  # binary
+  - src: .build/c8y-configuration-plugin
+    dst: /usr/bin/
+
```

Comment:
I'm assuming this is the default, as it was working fine without it earlier, and I don't see the same definition in the nfpm definitions of other packages.

Reply:
The link to the binary was removed a few weeks ago when we made the switch to the multi-binary setup. However this PR replaces the previous c8y-configuration-plugin with tedge-configuration-plugin (inside the tedge binary). And since the c8y-configuration-plugin is no longer included in the multi-call binary, it needs to be manually added to the package.